packages/react/src/components/field/field.tsx

```diff
@@ -1,12 +1,61 @@
-import { c, classy } from '@onfido/castor';
-import React from 'react';
+import { c, classy, FieldProps as BaseProps, FieldState } from '@onfido/castor';
+import { useForm } from '@onfido/castor-react';
+import React, { SyntheticEvent, useEffect, useState } from 'react';
+import { FieldProvider } from './useField';
+
+export { useField } from './useField';
 
 /**
  * Intended to wrap `Input` and `Textarea` components, so that `FieldLabel` and
  * `Validation` can wrap nicely following the minimum content allowed.
  */
-export const Field = ({ className, ...restProps }: FieldProps): JSX.Element => (
-  <div {...restProps} className={classy(c('field'), className)} />
-);
+export const Field = ({
+  className,
+  disabled,
+  onBlur,
+  onChange,
+  onInvalid,
```

Comment:
To align with all other components, keeping `className` below other props:

```suggestion
  disabled,
  onBlur,
  onChange,
  onInvalid,
  className,
```